site/app/templates/superuser/SamlManager.twig

```diff
@@ -21,36 +21,34 @@
     <button class="btn btn-primary" id="new-proxy-mapping-btn">Assign Proxy Mapping</button>
     <form method="POST" action="{{ validate_url }}" style="display: inline;">
         <input type="hidden" name="csrf_token" value="{{ csrf_token }}">
-        <button class="btn btn-warning">Validate Mapped Users</button>
+        <button class="btn btn-primary">Validate Mapped Users</button>
     </form>
     <br><br>
     <h2>Mapped Proxy Users</h2>
-    <table class="table">
+    <table class="table" style="text-align:left;">
         <tr>
             <th>SAML Username</th>
             <th>Submitty Username</th>
-            <th>Active?</th>
-            <th>Action</th>
+            <th style="text-align:center;">Action</th>
         </tr>
         {% for row in proxy_mapped_users %}
         <tr>
             <td>{{ row['saml_id'] }}</td>
             <td>{{ row['user_id'] }}</td>
-            <td>{{ row['active'] ? 'True' : 'False' }}</td>
-            <td>
+            <td style="text-align:center;">
                 {% if row['active'] %}
                     <form method="POST" action="{{ change_active_saml_url }}" style="display: inline;">
                         <input type="hidden" name="csrf_token" value="{{ csrf_token }}">
                         <input type="hidden" name="activate" value="0">
                         <input type="hidden" name="id" value="{{ row['id'] }}">
-                        <button class="btn btn-warning">Deactivate</button>
+                        <button class="btn btn-default">Deactivate</button>
                     </form>
                 {% else %}
                     <form method="POST" action="{{ change_active_saml_url }}" style="display: inline;">
                         <input type="hidden" name="csrf_token" value="{{ csrf_token }}">
                         <input type="hidden" name="activate" value="1">
                         <input type="hidden" name="id" value="{{ row['id'] }}">
-                        <button class="btn btn-warning">Activate</button>
+                        <button class="btn btn-primary" style="padding: 6px 21px">Activate</button>
```

Comment:
This padding shouldn't be set manually like this.  I'm not certain what the intent is, but doing this is prone to conflicts in the future, given that the button already has padding set.  If we were to change the styling of the `btn`  class, this particular button would be left behind.